root/components/common-macros.tt

```diff
@@ -350,8 +350,12 @@ END -%]
     IF namevar AND action == 'show' AND content != ''
             AND (noescape ? html_unescape(content.remove('</?span\b[^>]*>')) : content) != entity.name;
         options.unshift('name_variation');
-        hover = hover != '' ? l('{name} – {additional_info}', { name => entity.name, additional_info => hover })
-                            : entity.name;
+        IF (entity.sort_name && entity.sort_name == entity.name);
```

Comment:
Meaning if `entity.sort_name ` is literally the string `"0"`, it'll be "false" because perl. I think the intent here is to guard against entities that don't have a `sort_name` field at all, so it should be no longer needed if the check is moved as suggested below.